docs/task-with-secrets.md

```diff
@@ -156,17 +156,17 @@ vim task-secrets.yaml
 version: 1.0-preview-1
```

Comment:
Can you replace `1.0-preview-1` in all docs and exampels?

Reply:
Good catch, fixed